src/color-system.mdx

```diff
@@ -0,0 +1,98 @@
+
+Thus to get a solid blue color, you would write this: `color: rgb(var(--color-blue-default));`,
+or like this to get a blue color with 10% opacity: `color: rgba(var(--color-blue-default), 0.1);`
+
+### Available palettes
+In your web-components, you only need to import one of the palettes. Importing a palette makes all included CSS variables available in that web-component. It all depends on how many colors you need for that specific component. If you need more colors than what is offered in the UI palette, then simply remove that and import one of the other palettes that suits.
+
+Each palette also includes Lime Technologies' brand colors. So no matter which one you import, you will always have access to our brand colors and can use them by their specific CSS variable names.
+
+#### **UI palette** (default)
+Includes 10 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+
+<limel-example-ui-color-palette />
+
+In most cases, importing the UI palette is enough and gives you colors which you normally need in a user interface. This palette includes a few hand picked swatches that add an extra layer of meaning to different elements. Using these, an element in the UI can quickly communicate to the user what it does or indicates.
+
+As you notice, for each hue there is a `-default` and `-light` swatch available in this palette. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
+___
+
+#### **Primary palette**
+Includes 40 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+<limel-example-primary-color-palette />
+
+Sometimes, just a handful of colors is not enough and you need a bit more. For example when you need colors for your icons, or diagrams. This palette is designed to provide a decent number of harmonic colors while easing the choice of colors for you. Used for primary purposes and meaningful communications with colors.
+
+Note that this palette also includes brand colors, and contrast swatches.
+___
+
+#### **Extended palette**
+Includes 100 colors swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies' brand colors.
+
+<limel-example-extended-color-palette />
+
+This palette offers a much richer set of colors that can be used for visual communications, iconography, illustrations, and charts.
+
+___
+## Lime Technologies' Brand Colors
+<limel-example-brand-color-palette />
+
+These set of colors are included in all available palettes, and can be used for branding and marketing purposes. Swatches and their respective CSS variables have specific names starting with the prefix of `--lime-brand-color-`. All of these hues already exists in our color palette and mostly reside in the "dark mode" subset -as stated beside their names.
```

Comment:
> All of these hues already exist in our color palette and mostly reside in the "dark mode" subset -as stated beside their names.

I'm pretty confident I understand what you mean, but I'm worried this will confuse more people than it will enlight…

I think it might be better to remove this sentence from here, and add something about it on line 88 where dark mode support is mentioned. (See comment on that line.)

Reply:
sounds good